scene/gui/color_picker.h

```diff
@@ -128,6 +128,7 @@ class ColorPicker : public VBoxContainer {
 	int current_slider_count = SLIDER_COUNT;
 	static const int MODE_BUTTON_COUNT = 3;
 	const float WHEEL_RADIUS = 0.42;
+	const Vector2 PICKER_WINDOW_OFFSET_DECAL = Vector2(28, 28);
```

Comment:
There's a bunch more hardcoded values like this in ColorPicker, like `Vector2i(55, 72)` in the `else` branches where `28, 28` is used. It seems a bit weird to special case this one.

Reply:
I also suggest removing it, unless it will be used to correctly change the popup offset.

Reply:
Indeed, it probably better to remove it, it's not related to the fix, and if we want to use constants instead of hardcoded sizes/offsets the rest of the values should be changed as well.

Reply:
I just improve the code quality of he code I change. I don't touch something that works without valid reason. Having magic numbers is a bad practice, and should be avoided. Even if the rest of the code keep using magic numbers. It's my point, and I'm ok with this PR not merged if you disagree.